src/cli/commands/file/ls.js

```diff
@@ -11,6 +11,8 @@ module.exports = {
 
   handler (argv) {
     let path = argv.key
+    // `ipfs file ls` is deprecated. See https://ipfs.io/docs/commands/#ipfs-file-ls
+    print(`This functionality is deprecated, and will be removed in future versions. If possible, please use 'ipfs ls' instead.`)
```

Comment:
We should be suggesting `files ls`, right? per https://github.com/ipfs/specs/issues/98#issuecomment-241261107
copied this from ipfs.io, will change.

Reply:
Still just `ipfs ls` for now until both APIs are united. Currently, `ipfs ls !== ipfs files ls`

Reply:
Yeah I was a bit confused here. Tracing code, I found that `ipfs ls` is an alias of `ipfs files ls` in js-ipfs/core here - https://github.com/ipfs/js-ipfs/blob/905bdc03edfba23d35aa6a5b8e356ab56f6f36b1/src/core/index.js#L118

There is also no `/files/ls` route handler in js-ipfs and a `jsipfs files ls` command with a daemon running triggers a js-ipfs-api '/ls' request. So there are a couple places where it seems `ipfs ls == ipfs files ls`. I'm sure there's something that I'm missing, let's go over it Wednesday.